### ISSUE TITLE
Make elm-test-rs runnable directly through npx without needing elm-tooling setup

### ISSUE THREAD

**Issue by jjant** [mpizenberg/elm-test-rs#114]
Basically the title, it's a bit annoying having to set up another package management tool for this, I would like to just run `npx elm-test-rs` and have this working.
Thanks

**Comment by mpizenberg** [comment]
I'm not a JS user so almost never use npx, and don't want to maintain an
npm package. Is having an npm package the only option to be able to run it
with npx without having to install elm-tooling?

If so, since I'd like to avoid maintaining an npm package, do you think of
other options that would work better for you than the current solution?

**Comment by mpizenberg** [comment]
Another tool I was thinking of would be some kind of equivalent of npx,
specifically tailored for the elm ecosystem. It could be called "etx" for
elm tooling execute. But even if I have many ideas for it, I don't have any
time to work on this ^^ so etx is not an option for now.

**Comment by mpizenberg** [comment]
I've heard though that support for executables with npm is going to be much
easier, so if that is the case and someone wants to take a shot at it, I'm
not against it. But only if it doesn't mean I'll have update it every few
weeks because of security issues of some dependencies.

**Comment by avh4** [comment]
FYI I made the [`binwrap`](https://www.npmjs.com/package/binwrap) package to make npm distribution for elm-format as easy as possible.  You just need a file like this https://github.com/avh4/elm-format/blob/main/package/npm/index.js that says where the binaries are for each supported platform, and package.json like this https://github.com/avh4/elm-format/blob/main/package/npm/package.json  Unfortunately it does still get complaints about out-of-date dependencies, but I just ignore those and let folks download the binaries directly if they're concerned about that.

It's still not 100% ideal, but it's the best I've found so far (at least given I have a goal of supporting npm).
